.storybook/babel.config.js

```diff
@@ -0,0 +1,17 @@
+/*eslint-env node*/
+module.exports = {
```

Comment:
should we import and use our main babel config?

Reply:
yes, good call